### PR TITLE
Mss 218 parameter configuration

### PR DESCRIPTION
**Pull request by NathanielBennett**
Update the two Lambas that handle content to get their configs from ssm

### CODE REVIEW

File: cfn.yaml [hunk 1]
```diff
@@ -53,6 +53,12 @@ Resources:
                 - lambda.amazonaws.com
       Path: /
       Policies:
+        - PolicyName: assume-mobile-ssm-read-config-role
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action: sts:AssumeRole
+              Resource: !Sub arn:aws:iam::${MobileAccountId}:role/${CrossAccountBaseRoleName}-ssm-${Stage}
         - PolicyName: assume-mobile-s3-read-role
```
Comment (alexduf) [inline cfn.yaml]: shouldn't this one be removed?

File: riff-raff.yaml [hunk 2]
```diff
@@ -9,3 +9,10 @@ deployments:
       functionNames: [mobile-content-notifications-lambda-liveblogs-, mobile-content-notifications-lambda-]
       fileName: mobile-content-notifications-lambda.jar
       prefixStack: false
+   mobile-content-notifications-lambda-cfn:
```
Comment (alexduf) [inline riff-raff.yaml]: I seem to remember this wasn't working as the `riffraff` user in the CAPI account doesn't give us the right to do some of the operations. Has that changed?

File: cfn.yaml (hunk 1 above)
Comment (NathanielBennett) [reply cfn.yaml]: Yup thought I had .. :(

File: riff-raff.yaml (hunk 2 above)
Comment (NathanielBennett) [reply riff-raff.yaml]: It seemed like it work when I just tried it. Otherwise it's a case of manually updating the stack ..

**Pull request merged**